bindings/protocols/bacnet/index.html

```diff
@@ -1,1192 +1,1751 @@
+                                        <li>ackReq (opt.)</li>
+                                        <li>toState</li>
+                                        <li>origSource (opt.)</li>
+                                        <li>eventType (opt.)</li>
+                                        <li>values (opt.)</li>
+                                        <li>message (opt.)</li>
+                                        <li>inactive (opt.)</li>
+                                    </ul>
+                                </li>
+                                <li>moreEvents: Flag indicating if more events should be queried</li>
+                            </ul>
+                        </td>
+                        <td>
+                            <ul>
+                                <li>
+                                    List of Event Summaries:
+                                    <ul>
+                                        <li>Object Identifier</li>
+                                        <li>Event State</li>
+                                        <li>Acknowledged Transitions</li>
+                                        <li>Event Timestamps</li>
+                                        <li>Notify Type</li>
+                                        <li>Event Enable</li>
+                                        <li>Event Priorities</li>
+                                    </ul>
+                                </li>
+                                <li>More Events</li>
+                            </ul>
+                        </td>
+                        <td>
+                            As indicated above, the BACnet payload and the ActionAffordance output differ especially around how events are grouped.
+                            In the BACnet payload, one list item represents one object with its three possible events, which is reflected in Event State, Acknowledged Transitions, Event Timestamps and Priorities.
+                            This is to be processed and split into one standalone event for every current event of this object, or past unacknowledged event, with corresponding timestamp, priority...
+                        </td>
+                    </tr>
+                    <tr>
+                        <td>AcknowledgeEvent: ActionAffordance.input</td>
+                        <td>
+                            <ul>
+                                <li>eventSource: Source of the event to be acknowledged</li>
+                                <li>eventState: To-state of the event to be acknowledged</li>
+                                <li>eventTime: Timestamp of the event to be acknowledged</li>
+                                <li>eventTime: Timestamp of the event to be acknowledged</li>
+                                <li>ackSource: String describing the acknowledgement source</li>
+                                <li>ackTime: Timestamp of the acknowledge operation</li>

```

Comment:
```suggestion
                        <td>AcknowledgeEvent — ActionAffordance.input</td>
                        <td>
                            <ul>
                                <li>eventSource — Source of the event to be acknowledged</li>
                                <li>eventState — To-state of the event to be acknowledged</li>
                                <li>eventTime — Timestamp of the event to be acknowledged</li>
                                <li>eventTime — Timestamp of the event to be acknowledged</li>
                                <li>ackSource — String describing the acknowledgement source</li>
                                <li>ackTime — Timestamp of the acknowledge operation</li>
```

Reply:
I use the colon character to indicate I am defining the meaning of these terms, like a vocabulary notation. Is there a specific reason to use the dash instead?

Reply:
It's a stronger visual separator. Also, a number of tools (wrongly) include the colon when double-clicking to select the term, but they don't include the em-dash (because it's separated by a space). (I believe this select behavior is part of why programmers often insert a space before a colon, which space is itself incorrect.)